js/ascendex.js

```diff
@@ -1266,7 +1267,7 @@ module.exports = class ascendex extends Exchange {
         if (symbol !== undefined) {
             market = this.market (symbol);
         }
-        const defaultAccountCategory = this.safeString (this.options, 'account-category');
+        const defaultAccountCategory = this.safeString (this.options, 'account-category', this.defaultType);
```

Comment:
There's no such thing as `this.defaultType`, but there is `this.options['defaultType']` and it's already handled in handleMarketTypeAndParams: 

- https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L2131
- https://github.com/ccxt/ccxt/blob/master/python/ccxt/base/exchange.py#L2720
- https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L3599